signac/__main__.py

```diff
@@ -1045,10 +1045,13 @@ def jobs():
                     try:
                         readline.read_history_file(fn_hist)
                         readline.set_history_length(1000)
-                    except (IOError, OSError) as error:
-                        if error.errno != errno.ENOENT:
-                            raise
-                    atexit.register(readline.write_history_file, fn_hist)
+                        with open(fn_hist, 'w') as f: pass
+                        atexit.register(readline.write_history_file, fn_hist)
+                    except FileNotFoundError:
+                        atexit.register(readline.write_history_file, fn_hist)
+                    except PermissionError:
+                        print("Warning: .signac_shell_history does not have read/write permission. "
```

Comment:
I'd like to replace the literal `.signac_shell_history` with `os.path.relpath(fn_hist)` instead so that we have only one single source of truth with respect to this filename.

Reply:
@csadorf changed it.